std-bits/table/src/main/java/org/enso/table/expressions/ExpressionVisitorImpl.java

```diff
@@ -166,6 +186,25 @@ private Value executeMethod(String name, Value... args) {
     }
   }
 
+  private Object[] prepareArguments(Method method, Value[] args) {
+    Object[] objects;
+    if (method.isVariableArgumentMethod()) {
+      objects = new Object[2];
+      objects[0] = wrapAsColumn(args[0]);
+      objects[1] = Arrays.copyOfRange(args, 1, args.length, Object[].class);
+    } else if (method.isStaticMethod()) {
+      // The static method takes the type as the synthetic 'self' argument, so we need to prepend
+      // it:
+      objects = new Object[args.length + 1];
+      objects[0] = method.getStaticsType();
+      System.arraycopy(args, 0, objects, 1, args.length);
+    } else {
+      objects = Arrays.copyOf(args, args.length, Object[].class);
+      objects[0] = wrapAsColumn(args[0]);
+    }
+    return objects;
+  }
```

Comment:
I feel like exposing `getStaticsType` outside of Method is a bit leaking an implementation detail, no?

Wouldn't it make more sense to move the `prepareArguments` method into the `Method` class? As it seems to be quite tightly coupled with `Method`.

Reply:
Yes I think you are right. I originally started with having execute on Method, but there was a reason that didn't work. But moving prepareArguments in I think should work.